lib/tools/system-calls.js

```diff
@@ -311,6 +311,26 @@ systemCallMethods.killProcess = async function (process, opts = {}) {
   }
 };
 
+systemCallMethods.push = async function (file, dest) {
```

Comment:
already has push and pull method.
check sdb-commands.js

Reply:
I'll close this PR.